src/sagemaker/jumpstart/curated_hub/utils.py

```diff
@@ -164,3 +179,97 @@ def create_hub_bucket_if_it_does_not_exist(
     )
 
     return bucket_name
+
+def tag_hub_content(hub_content_arn: str, tags: List[Tag], session: Session) -> List[Dict[str, List[Dict[str, str]]]]:
```

Comment:
This is where typings get confusing if you use a generic type that doubles as the real typing required by the SDK.. If I'm a contributor, I think that I can just pass `tags` straight into `add_tags` because of the type. This isn't the case though. See my earlier comment on removing this typing completely and then you can optionally use the one from the SDK so it's consistent